composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "equip/console",
+    "description": "Equip console",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Equip Contributors",
+            "homepage": "https://github.com/equip"
+        }
+    ],
+    "require": {
+        "equip/config": "^1.0",
+        "equip/structure": "^1.1",
+        "php": ">=5.5",
```

Comment:
Despite the alphabetical organization, I think this line should always go first.